conkit/core/contactmap.py

```diff
@@ -682,9 +696,60 @@ def discard(*args, **kwargs):
 
         return contact_map
 
-    def match(
-        self, other, add_false_negatives=False, match_other=False, remove_unmatched=False, renumber=False, inplace=False
-    ):
+    def match_naive(self, other, add_false_negatives=False, match_other=False, inplace=False):
+        """Modify both hierarchies so residue numbers match one another
+
+        This function performs a naive match. It assumes the numbering in both
+        contact maps is equivalent and it will not attempt to perform a sequence
+        alignment.
+
+        Parameters
+        ----------
+        other : :obj:`~conkit.core.contactmap.ContactMap`
+           A ConKit :obj:`~conkit.core.contactmap.ContactMap`
+        add_false_negatives : bool
+           Add false negatives to the `self`, which are contacts in `other` but not in `self`
+
+           Required for :meth:`~conkit.core.contactmap.ContactMap.recall` and can be undone
+           with :meth:`~conkit.core.contactmap.ContactMap.remove_false_negatives`
+        match_other: bool, optional
+           Match `other` to `self` [default: False]
+        inplace : bool, optional
+           Replace the original contacts [default: False]
+
+        Returns
+        -------
+        :obj:`~conkit.core.contactmap.ContactMap`
+            :obj:`~conkit.core.contactmap.ContactMap` instance, regardless of inplace
+
+        """
+        contact_map1 = self._inplace(inplace)
+        if match_other:
+            contact_map2 = other._inplace(inplace)
+        else:
+            contact_map2 = other._inplace(False)
```

Comment:
If inplace is False, match_other doesn't actually do anything. Is this intended?

Reply:
Yes I agree this is a bit strange but this reproduces the behaviour of the original `conkit.core.ContactMap.match()` function so I left it as it is.